### ISSUE TITLE
HSV colorspace support.

### ISSUE THREAD

**Issue by phoe** [slyrus/opticl#30]
I have a usecase where I want to perform hue rotation on images. This requires that the image is converted into HSV colorspace, which I couldn't find in opticl yet.

**Comment by slyrus** [comment]
Patches welcome! :)

**Comment by phoe** [comment]
Yep - as soon as I have some time, I'll happily contribute, perhaps cannibalizing some code from https://github.com/wmannis/simple-rgb or https://github.com/privet-kitty/dufy . I put this issue here mostly so I don't forget.

**Comment by slyrus** [comment]
@phoe It's a little complicated to use as you need the hsv-image branch of opticl and opticl-core, and the example code is opticl-more-test (on the hsv-image branch) but I was able to rotate the hue in the example and  convert to/from 8-bit-rgb-image.
